girder/api/describe.py

```diff
@@ -540,6 +540,20 @@ def _passArg(self, fun, kwargs, name, val):
         else:
             kwargs['params'][name] = val
 
+    def _mungeKwargs(self, kwargs, fun):
+        """
+        Performs final modifications to the kwargs passed into the wrapped function.
+        Combines the sort/sortdir params appropriately for consumption by the model
+        layer, and only passes the "params" catch-all dict if there is a corresponding
+        kwarg for it in the wrapped function.
+        """
+        if self.description.hasPagingParams and 'sort' in kwargs:
+            sortdir = kwargs.pop('sortdir', None) or kwargs['params'].pop('sortdir', None)
+            kwargs['sort'] = [(kwargs['sort'], sortdir)]
+
+        if 'params' not in fun._fnArgs and not fun._fnKeywds:
+            kwargs.pop('params', None)
```

Comment:
It's a dict that will contain any parameters that were passed but were not listed in the description. Is that the question you were asking...?